ct/actualbudget.sh

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env bash
+source <(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVED/main/misc/build.func)
+# Copyright (c) 2021-2025 community-scripts ORG
+# Author: MickLesk (CanbiZ)
+# License: MIT | https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+# Source: https://actualbudget.org/
+
+APP="Actual Budget"
```

Comment:
```suggestion
APP="Actual-Budget"
```
Dash needed here as i think about it?

Reply:
do the system himself

Reply:
ok.